dcspy/sdk/__init__.py

```diff
@@ -33,7 +33,7 @@ class DllSdk:
 KeyDll = DllSdk(name='Gkey', dir='G-key', header=key_header)
 
 
-def load_dll(lib_type: DllSdk) -> Optional[Lib]:
+def load_dll(lib_type: DllSdk) -> Optional[Lib | CDLL]:
```

Comment:
This, and it's usage, and causing some linting issues which I'm not sure how to resolve. e.g.

`dcspy\sdk\key_sdk.py:40: error: Incompatible types in assignment (expression has type "Lib | CDLL | None", variable has type "CDLL")  [assignment]`

Reply:
1. Yeah `mypy` sometimes is pain in.... 
2. `|` is not supported by Python 3.10+ this need be change into `Uinion`